UPGRADE-1.8.md

```diff
@@ -1,10 +1,19 @@
-# UPGRADE FROM `v1.8.4` TO `v1.8.5`
+# UPGRADE FROM `v1.8.4` TO `v1.8.6`
 
 1. Change configuration of new ApiBundle in your `config/packages/security.yaml` file:
 
     ```diff
-        security:
-            access_control:
+         security:
+             firewalls:
+                 new_api_admin_user:
+                     json_login:
+    -                    check_path: "%sylius.security.new_api_route%/admin/authentication-token"
+    +                    check_path: "%sylius.security.new_api_admin_route%/authentication-token"
+                 new_api_shop_user:
+                     json_login:
+    -                    check_path: "%sylius.security.new_api_route%/shop/authentication-token"
+    +                    check_path: "%sylius.security.new_api_shop_route%/authentication-token"
+             access_control:
```

Comment:
Maybe it would be better to introduce it in `1.8.5` -> `1.8.6` upgrade note? But I don't know isn't it too finicky 💃

Reply:
Yeah, I thought about it… and had the same hesitation 😅

Reply:
This should be in 1.8.5 actually… it's a mistake not to have this change in the previous PR :p.

That's why I hesitate. Let me know about it and I'll do the change if needed. It's not complicated :p

Reply:
@Zales0123
Do I change? Or not? What do you want me to do?

Reply:
Sorry for missing this comment, let it be as it is 😄